tests/main/interfaces-browser-support/task.yaml

```diff
@@ -0,0 +1,138 @@
+summary: check that the browser-support interface works
```

Comment:
Total nitpick: I think our summary strings in spread tests all start with captial letter.